src/shared/service/seederService.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { DataSource } from 'typeorm';
+
+@Injectable()
+export class SeederService {
```

Comment:
This service should be included in the `test` folder. Under `shared` we should, for example, have some util functions used by the other modules.

Reply:
will do if enough time remains